dev-python/radon/radon-4.1.0.ebuild

```diff
@@ -0,0 +1,50 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+PYTHON_COMPAT=( python3_{6,7,8} )
+
+inherit distutils-r1
+
+DESCRIPTION="Code Metrics in Python"
+HOMEPAGE="https://radon.readthedocs.org/"
+SRC_URI="mirror://pypi/${P:0:1}/${PN}/${P}.tar.gz"
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="doc test"
+RESTRICT="!test? ( test )"
+
+CDEPEND="dev-python/setuptools[${PYTHON_USEDEP}]"
+RCDEPEND="
+	dev-python/colorama[${PYTHON_USEDEP}]
+	dev-python/flake8[${PYTHON_USEDEP}]
```

Comment:
We'll also need to update PYTHON_COMPAT in dev-python/flake8-polyfill and and add it as a test dependency, since the unit tests are failing like this:
```
__________________________________________________________________________ ERROR collecting radon/tests/test_cli.py __________________________________________________________________________
ImportError while importing test module '/var/tmp/portage/dev-python/radon-4.1.0/work/radon-4.1.0/radon/tests/test_cli.py'.
Hint: make sure your test modules/packages have valid Python names.
Traceback:
radon/tests/test_cli.py:7: in <module>
    import radon.cli as cli
radon/cli/__init__.py:10: in <module>
    import radon.complexity as cc_mod
radon/complexity.py:6: in <module>
    from flake8_polyfill import options
E   ModuleNotFoundError: No module named 'flake8_polyfill'
```

Reply:
Fixed. dev-python/pytest-mock was also missing.